frontend/src/components/TournamentTeam.js

```diff
@@ -32,7 +35,7 @@ const TournamentTeam = () => {
   const [doneFetching, setDoneFetching] = createSignal(false);
 
   const tournamentQuery = createQuery(
-    () => ["tournaments", params.tournament_slug],
+    () => ["tournament", params.tournament_slug],
```

Comment:
We are using "tournaments" for query name everywhere
![Screenshot 2024-07-13 at 4 05 37 PM](https://github.com/user-attachments/assets/94d75968-5587-4a47-a641-aec501d44112)